xbmc/pvr/recordings/PVRRecording.cpp

```diff
@@ -363,7 +380,7 @@ void CPVRRecording::UpdatePath(void)
       strDirectory = StringUtils::Format("%s/", m_strDirectory.c_str());
     if (!m_strChannelName.empty())
       strChannel = StringUtils::Format(" (%s)", m_strChannelName.c_str());
-    m_strFileNameAndPath = StringUtils::Format("pvr://recordings/%s%s, TV%s, %s.pvr", strDirectory.c_str(), strTitle.c_str(), strChannel.c_str(), strDatetime.c_str());
+    m_strFileNameAndPath = StringUtils::Format("pvr://recordings%s/%s%s, TV%s, %s.pvr", (m_isDeleted ? "-deleted" : ""),  strDirectory.c_str(), strTitle.c_str(), strChannel.c_str(), strDatetime.c_str());
   }
```

Comment:
I'm not sure if we should use a suffix instead of a subdir "pvr://recordings/deleted/" and "pvr://recordings/active/".